container-search/src/main/java/com/yahoo/search/handler/SearchHandler.java

```diff
@@ -577,14 +566,14 @@ public void createRequestMapping(Inspector inspector, Map<String, String> map, S
                     map.put(qualifiedKey , value.asString());
                     break;
                 case ARRAY:
-                    map.put(qualifiedKey, value.asString());
+                    map.put(qualifiedKey, value.toString()); // XXX: Causes parsing the JSON twice (Query.setPropertiesFromRequestMap)
```

Comment:
👍 .
Weird that I did not notice this bug while testing... 😅
In any case, it is good that it gets fixed now.